Makefile

```diff
@@ -20,7 +20,7 @@ DEST = project
 PYMODEL = $(SCHEMA_NAME)
 EXAMPLEDIR = examples
 
-.PHONY: all clean combined-extras examples-clean site test
+.PHONY: all clean site-materialized examples-clean site test
```

Comment:
site-materialized has been removed. can remove all references elsewhere in the Makefile